packages/patternfly-4/react-core/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/Alert/alert.css';
+import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import { AlertIcon } from './helpers/AlertIcon';
+import { capitalize } from '../../helpers/util';
+import { Omit } from '../../helpers/typeUtils';
+
+export enum AlertVariant {
+  success = 'success',
+  danger = 'danger',
+  warning = 'warning',
+  info = 'info'
+}
+
+export interface AlertProps
+  extends Omit<React.HTMLProps<HTMLDivElement>, 'action' | 'title'> {
+  /** Adds Alert variant styles  */
+  variant: keyof typeof AlertVariant;
+  /** Title of the Alert  */
+  title: React.ReactNode;
+  /** Action button to put in the Alert.  Should be <AlertActionLink> or <AlertActionCloseButton>  */
+  action?: React.ReactNode;
+  /** content rendered inside the Alert  */
+  children?: React.ReactNode;
+  /** additional classes added to the Alert  */
+  className?: string;
+  /** Adds accessible text to the Alert */
+  'aria-label'?: string,
+  /** Variant label text for screen readers  */
+  variantLabel?: string;
+};
+
+export const Alert = ({
+  variant,
+  variantLabel = null,
+  'aria-label': ariaLabel = `${capitalize(variant)} Alert`,
+  action = null,
+  title,
+  children = '',
+  className = '',
+  ...props
+}: AlertProps) => {
+  variantLabel = variantLabel || capitalize(variant);
```

Comment:
They aren't the same. The current logic is if a consumer passes a `variantLabel`, we want to use it instead of `capitalize(variant)`.

Reply:
I just tried it, and it results in the same output.  
```suggestion
export const Alert: React.FunctionComponent<AlertProps> = ({
  variant,
  variantLabel = capitalize(variant),
  'aria-label': ariaLabel = `${capitalize(variant)} Alert`,
  action = null,
  title,
  children = '',
  className = '',
  ...props
}: AlertProps) => {
  const readerTitle = (
    <React.Fragment>
      <span className={css(accessibleStyles.screenReader)}>{`${variantLabel} alert:`}</span>
      {title}
    </React.Fragment>
  );
```
Also it makes the document correct instead of saying it defaults to null.
<img width="603" alt="Screen Shot 2019-05-14 at 12 07 51 PM" src="https://user-images.githubusercontent.com/6126356/57713965-776b8e00-7641-11e9-8fa7-18507ae5f765.png">

Reply:
Whoops, you're right. I thought the line read `variant = variantLabel || capitalize(variant);`